components/home/modules/ROOT/pages/index.adoc

```diff
@@ -1,30 +1,25 @@
 = Documentation
 
-Explore our guides and examples to integrate OpenZeppelin security products to all aspects of your Ethereum development and operations.    
+Explore our guides and examples to learn how to integrate OpenZeppelin security products with your project.    
 
 [.card-section]
 == Explore using OpenZeppelin
 
-OpenZeppelin provides a complete suite of security products to build, manage, and inspect all aspects of software development and operations for Ethereum projects.
+OpenZeppelin provides a complete suite of security products to adopt security best practices from the first line of code all the way to running your decentralized application on-chain.
 
 [.card.card-primary.card-contracts]
 --
 xref:contracts::index.adoc[[.card-title]#Contracts# [.card-body]#pass:q[A library of modular, reusable, and secure smart contracts, written in Solidity.]#]
 --
 
-[.card.card-primary.card-upgrades-js]
---
-xref:upgrades.adoc[[.card-title]#Upgrades# [.card-body]#pass:q[A comprehensive suite of contracts and tooling to deploy and manage upgradeable contracts on Ethereum.]#]
---
-
 [.card.card-primary.card-defender]
 --
-xref:defender::index.adoc[[.card-title]#Defender# [.card-body]#pass:q[A secure platform to automate Ethereum operations.]#]
+xref:defender::index.adoc[[.card-title]#Defender# [.card-body]#pass:q[A secure platform to code, audit, deploy, monitor, and operate your blockchain application.]#]
 --
 
-[.card.card-secondary.card-test-helpers]
```

Comment:
I removed Test Helpers. We're not actively maintaining them and they should not be featured prominently.